docs/mastering-plone/volto_frontpage.md

```diff
@@ -57,14 +57,14 @@ To understand why we need a collection criterion for a dynamic frontpage in Volt
 :align: left
 ```
 
-In the sidebar we see the `criteria` selection and if we click there, it'll show some of the selectable criterions ordered in categories like the following:
+In the sidebar we see the `criteria` selection and if we click there, it'll show some of the selectable criteria ordered in categories like the following:
```

Comment:
For UI bits and pieces, we should use semantic markup with `guilabel`, instead of inline literal markup which is usually used for inline commands or code.

```suggestion
In the sidebar, we see the {guilabel}`Criteria` select menu, and if we click there, it'll show some of the selectable criteria ordered in categories like the following:
```